src/poliastro/bodies.py

```diff
@@ -19,17 +19,56 @@
 
 Data references can be found in :py:mod:`~poliastro.constants`
 """
+import math
+from functools import wraps
```

Comment:
If you are not using the decorator anymore, I would remove it along with this import!